packages/react-devtools-shared/src/__tests__/profilingCache-test.js

```diff
@@ -716,4 +716,93 @@ describe('ProfilingCache', () => {
       TestRenderer.create(<Validator commitIndex={0} rootID={rootID} />);
     });
   });
+
+  // See https://github.com/facebook/react/issues/18831
+  it('should not crash during route transitions with Suspense', () => {
```

Comment:
This newly added test caught the regression between 16.13.1 (NPM release) and previous DevTools master.